demo-apps/angular/angular-app/src/styles.css

```diff
@@ -0,0 +1,9 @@
+/* You can add global styles to this file, and also import other style files */
+@import '../node_modules/@cdssnc/gcds-components/dist/gcds/gcds.css';
+
+/* Reset Styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
If we add the UF, these reset styles can be removed as they will be added by the UF.